playbooks/nvidia-egx/egxstack-installation.yml

```diff
@@ -1,4 +1,4 @@
-- include: k8s-cluster.yml
```

Comment:
I can't figure out how to leave a comment on a renamed file. :)

We moved nvidia-mig.yml into /nvidia-dgx. But that playbook will be relevant to any server running an NVIDIA A100, not just DGX devices. I'm okay with keeping it in the nvidia-dgx directory and only "officially supporting" that playbook on DGX systems. what are your thoughts?

Reply:
Looking further, I see we have nvidia-software and utilities. Things like set GPU clock/GPU power are in the utilities folder. I might place this there as well.